src/libraries/System.Net.Requests/src/System.Net.Requests.csproj

```diff
@@ -3,8 +3,12 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser</TargetFrameworks>
     <Nullable>enable</Nullable>
+    <NoWarn Condition="'$(TargetOS)' == 'Browser'">$(NoWarn);CS0809</NoWarn>
```

Comment:
It's to avoid the build errors like
```
/Users/maxim/runtime/artifacts/obj/System.Net.Requests/net5.0-Browser-Release/System.Net.Requests.notsupported.cs(268,33): error CS0809: Obsolete member 'HttpWebResponse.GetObjectData(SerializationInfo, StreamingContext)' overrides non-obsolete member 'WebResponse.GetObjectData(SerializationInfo, StreamingContext)' [/Users/admin/dev/runtime/src/libraries/System.Net.Requests/src/System.Net.Requests.csproj]
```
The ref .csproj already has the same.

Reply:
Yeah that sounds good. However instead of making it specific for `Browser` I would make the PNSE generation resilient for this as we might end-up adding PNSE assembly for other platforms? Maybe? 

So probably set this in line `10` with something like:

```xml
<NoWarn Condition="'$(GeneratePlatformNotSupportedAssemblyMessage )' != ''">$(NoWarn);CS0809</NoWarn>
```

Reply:
Fixed, thanks!

Reply:
hm, it doesn't work

Reply:
@safern What did I do wrong? The build fails with the original error.

Reply:
it was redundant space :)